docs/admin/index.rst

```diff
@@ -94,105 +141,75 @@ Once all the pieces are in place, you can deploy Renku with the following comman
 
-To activate this feature, please set ``ui.privacy.enabled: true``. We have already configured a
-default cookie banner to inform the users about the aforementioned requirements and points to
-point them to the privacy page for further details.
+#. turn off automatic redirect to GitLab by setting redeploying with the value ``gitlab.oauth.autoSignIn: false``
+#. log in as the root user using the password from ``gitlab.password``
+#. modify any users you want to modify (e.g. to make them admin)
+#. turn the ``autoSignIn`` setting back on
 
-It's possible to customize the banner to your own needs, changing both the appearance and the
-content. We use `cookie consent <https://github.com/Mastermindzh/react-cookie-consent>`_ to
-create the banner and we support customization of all the properties exposed by the library.
-All the adjustments should be provided as values in ``ui.privacy.banner`` as follows:
+Independent GitLab as identity provider
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. ``content``: The HTML content of the cookie banner. Please mind that links should point
-   to local content. Use the privacy page to add references to third party sites.
-2. ``layout``: The list of properties for the cookies consent plugin. We use
-   `Bootstrap <https://getbootstrap.com/docs>`_ for the UI, therefore all the Bootstrap
-   classes can be used to customize the appearance.
+If the RenkuLab deployment is using an existing GitLab, this application could be configured as an identity provider for RenkuLab's keycloak. This way all of the users from the existing GitLab instance could authenticate.
 
-You can refer to the current
-`values.yaml in the renku-ui repository <https://github.com/SwissDataScienceCenter/renku-ui/blob/master/helm-chart/renku-ui/values.yaml>`_
-to check the default values and to find an example of how to apply different settings.
+#. Go to https://<renku-domain>/auth (or to your stand-alone Keycloak dashboard), login to ``Admin console`` using admin username and the decoded password stored in ``keycloak-password-secret``.
+#. Add an ``Identity Provider`` of type ```OpenID Connect v1.0``.
+#. Set ``Alias`` to <renku-domain>, ``Authorization URL`` to https://<gitlab-domain>/oauth/authorize , ``Token URL`` to https://<gitlab-domain>/oauth/token and ``Client ID`` and ``Client Secret`` to the values used in step 1.
+#. Click ``Save``. The new Identity Provider should appear and any user from the stand-alone GitLab instance should be able to login.
 
-Troubleshooting
+Updating RenkuLab
 ------------------
 
-If some Renku pods are not starting or present some errors please check the troubleshooting page.
+To update Renku to a newer version please check the **Upgrading from 0.x.x** section of the `release notes <https://github.com/SwissDataScienceCenter/renku/releases>`_ in case the renku-values.yaml file needs to be modified.
```

Comment:
```suggestion
To update RenkuLab to a newer version please check the **Upgrading from 0.x.x** section of the `release notes <https://github.com/SwissDataScienceCenter/renku/releases>`_ in case the renku-values.yaml file needs to be modified.
```